.github/workflows/ci.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
           - rust/workspace-crate
     steps:
       - uses: actions/checkout@v3
+        with:
+          submodules: recursive
```

Comment:
:question: This repo currently doesn't have submodules. I'm understanding this change as standardizing on recursing submodules being the default.??

Reply:
Yeah, if we didn't do this, we'd need to escape every `{{` in `ci.yaml`, which is pretty tedious, and I can't imagine a scenario where I'd want to checkout a repo managed by this template but not checkout its submodules.